collections/hub/provisioning_dfsp/newdfsp.json

```diff
@@ -8,6 +8,90 @@
         "info": "DFSP account"
       },
       "requests": [
+        {
+          "id": -2,
+          "meta": {
+            "info": "Add {$inputs.HUB_NAME} Account-HUB_MULTILATERAL_SETTLEMENT Second Currency"
+          },
+          "description": "Add {$inputs.HUB_NAME} Account-HUB_MULTILATERAL_SETTLEMENT Second Currency",
+          "apiVersion": {
```

Comment:
I think its better to move these into another test case and please try to avoid minus numbers. I don't know if it works with TTK UI. So the first test case is "id: 1, name: Setup Hub accounts" and second is "id:2,  name: Setup DFSP accounts and limits". And also please try to load the collection in TTK UI, and it is recommended to use TTK UI for creating / updating collections. Editing Json files is error prone.

Reply:
ok, done